packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -217,6 +221,15 @@ const styles: ComponentStyles<ISiteManagedClasses, IDevSiteDesignSystem> = {
         alignItems: "center",
         justifyContent: "center"
     },
+    site_transparencyToggleButton: {
```

Comment:
can we add "cursor: pointer" here to ensure all browsers show that this is actionable?

Reply:
@eljefe223 making sure you saw this last one